swayidle/swayidle.1.txt

```diff
@@ -0,0 +1,51 @@
+/////
+vim:set ts=4 sw=4 tw=82 noet:
+/////
+:quotes.~:
+
+swayidle (1)
+============
+
+Name
+----
+swayidle - Screen power manager for sway
+
+Synopsis
+--------
+'swayidle' [options] [timeout <seconds> exec <parameter>
+			[resume exec <parameter>]] ...
+
+Swayidle handles idle events from sway: it sets up the lockscreen and turns
+off the screen. It is also possible to execute arbitrary commands or send
+commands to sway, by passing a number of commands at the end of the
+commandline. Swayidle will do nothing by default, it will require some
+commands.
+
+Command syntax:
+[timeout <seconds> exec <parameter> [resume exec
+<parameter>]]
+
+It is possible to pass multiple commands to setup multiple idle handlers.
```

Comment:
This could be better at describing each command line "subcommand". I can take a crack at rewriting it if you like

Reply:
Its probably better, since my native tongue is Swedish...